GeoDaCenter/covid-v3
# [WIP] feat: add new "Stories" page

## Problem
We need a new page for the "Stories Showcase"

Starts to fix #82

## Approach
* feat: added "Stories Showcase" page located at `/stories`
    * Updated Navbar / Footer links to point to this new page

## How to Test
1. Navigate to the homepage
2. Expand the Stories section in the navbar, and click on Overview
    * You should be taken to the new "Stories Showcase" page
3. Navigate back to the homepage
4. Scroll down to the Footer and click on Stories
    * TBD - acceptance criteria for this new page
